src/questionnaireitem_media_video.js

```diff
@@ -40,11 +42,28 @@ QuestionnaireItemMediaVideo.prototype._createAnswerNode = function() {
 };
 
 QuestionnaireItemMediaVideo.prototype.releaseUI = function() {
-    this.node = null;
     this.uiCreated = false;
     this.enabled = false;
 
+    this.videoNode.pause();
+    if (Array.isArray(this.videoSource)) {
```

Comment:
I would recommend making this.videoSource an array by default in the constructor.
It's easier to read.